week5/day3/mutationObserver/src/components/Parent.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useRef, useEffect } from "react";
+import { ChildList } from "./ChildList";
+
+export const Parent = () => {
+  const [mounting, setMounting] = useState(true);
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    if (!containerRef.current || !mounting) return;
+
+    const config = {
+      attributes: true,
+      childList: true,
+      subtree: true,
+    };
```

Comment:
Можно вынести за хук и явно указать типизацию
```suggestion
    const config: MutationObserverInit = {
      attributes: true,
      childList: true,
      subtree: true,
    };
```